### PR TITLE
Cleanup: Correct some codestyle & remove/add whitespaces.

### PR DESCRIPTION
**Pull request by armano2**

### CODE REVIEW

**Comment by MitchesD** [comment]
please squash these commits or use better name for them to show what exactly it changed

**Comment by armano2** [comment]
ok.. :dancer:

File: src/server/game/Entities/Object/Object.h [hunk 1]
```diff
@@ -622,11 +622,11 @@ class WorldObject : public Object, public WorldLocation
...
-        uint32 urand(uint32 min, uint32 max) const  { return GetMap()->mtRand.randInt(max - min) + min;}
-        int32 rand32() const                        { return GetMap()->mtRand.randInt();}
-        double rand_norm() const                    { return GetMap()->mtRand.randExc();}
-        double rand_chance() const                  { return GetMap()->mtRand.randExc(100.0);}
+        int32 irand(int32 min, int32 max) const     { return int32(GetMap()->mtRand.randInt(max - min)) + min; }
+        uint32 urand(uint32 min, uint32 max) const  { return GetMap()->mtRand.randInt(max - min) + min; }
+        int32 rand32() const                        { return GetMap()->mtRand.randInt(); }
+        double rand_norm() const                    { return GetMap()->mtRand.randExc(); }
+        double rand_chance() const                  { return GetMap()->mtRand.randExc(100.0); }
 #endif
```
Comment (joschiwald) [inline src/server/game/Entities/Object/Object.h]: deprecated code?

Comment (armano2) [inline src/server/game/Entities/Object/Object.h]: yes this functions are no longer in use... and they are defined in Utils

**Comment by tkrokli** [comment]
@armano2 : Maybe you know this already, but your PR fails in Travis because your own repo is 10 commits behind TrinityCore:6.x and you have not implemented the commit from MitchesD, b01a9b93be .

**Comment by armano2** [comment]
yes i know, soon i will fix it

**Comment by MitchesD** [comment]
@armano2  please finish it (squash and rebase)

**Comment by armano2** [comment]
now i trying to clean up entire project from thinks like that :dancer:

**Comment by MitchesD** [comment]
I see no progress there, when you are done please open new Pull Request

**Pull request closed without merging**